src/shaders/ssao.frag

```diff
@@ -13,12 +13,18 @@ uniform mat4 projection;
 
 // scaling for the noise texture coords to ensure that the noise texture tiles across the screen.
 const vec2 noise_scale = vec2(SCREEN_WIDTH / 5.0, SCREEN_HEIGHT / 5.0);
-
+const float MAX_DISTANCE = -0.5;
 
 
 void main()
 {
     vec3 frag_pos = texture(g_position, TexCoord).xyz;
+
+    if (frag_pos.z > MAX_DISTANCE) {
+        FragColor = 1.0;
+        return;
+    }
+
```

Comment:
This doesn't really add anything performance-wise since both branches are still "executed" - but I agree with the change anyway.